doc/decoding/decoding_tutorial.rst

```diff
@@ -187,28 +189,28 @@ In scikit-learn, the prediction objects have two important methods:
 
 .. warning::
 
-    **Do not predict on data used by the fit:** the prediction that we obtain here
-    is to good to be true (see next paragraph). Here we are just doing a sanity
-    check.
+    **Do not predict on data used by the fit:** the prediction that we obtain
+    here is to good to be true (see next paragraph).
+    Here we were just doing a sanity check.
 
 .. for doctests (smoke testing):
     >>> from sklearn.svm import SVC
     >>> svc = SVC()
 
 Measuring prediction performance
----------------------------------
+--------------------------------
 
 Cross-validation
-.................
+................
 
 However, the last analysis is *wrong*, as we have learned and tested on
-the same set of data. We need to use a cross-validation to split the data
+the *same* set of data. We need to use a cross-validation to split the data
 into different sets, called "folds", in a `K-Fold strategy
-<http://en.wikipedia.org/wiki/Cross-validation_(statistics)#k-fold_cross-validation>`_.
+<https://en.wikipedia.org/wiki/Cross-validation_(statistics)#k-fold_cross-validation>`_.
 
 We use a cross-validation object,
 :class:`sklearn.cross_validation.KFold`, that simply generates the
-indices of the folds within a loop.
+indeces of the folds within a loop.
```

Comment:
Funny how [both indices and indexes](http://www.nasdaqomx.com/transactions/indexes/indexesorindices/) are valid, but I changed it to a non existing spelling... I will correct that.